README.md

```diff
@@ -0,0 +1,39 @@
+# Cobrowse Agent UI
+
+## Localization
+
+### Change active locale at runtime
+
+The project that's using the agent-ui can dictate which locale to use in our components. Below is an example of how to
+achieve this from an app that's also using `i18next` to handle localization.
+
+```js
+// i18n.js
+
+import i18n from 'i18next';
+import i18nAgentUI from 'cobrowse-agent-ui/dist/i18n';
```

Comment:
Please update here too to avoid confusion!

Reply:
Whoops. Done!